benchmarks/multiplatform/benchmarks/src/wasmJsMain/kotlin/main.wasmJs.kt

```diff
@@ -1,15 +1,72 @@
+@file:OptIn(ExperimentalJsExport::class)
+
 import kotlinx.browser.window
-import kotlinx.coroutines.MainScope
-import kotlinx.coroutines.launch
+import kotlinx.coroutines.*
 import org.w3c.dom.url.URLSearchParams
+import kotlin.js.Promise
+
+val jsOne = 1.toJsNumber()
 
-fun main() {
+fun main(args: Array<String>) {
+    if (isD8env().toBoolean()) {
+        mainD8(args)
+    } else {
+        mainBrowser()
+    }
+}
+
+fun mainBrowser() {
     val urlParams = URLSearchParams(window.location.search.toJsString())
     var i = 0
     val args = generateSequence { urlParams.get("arg${i++}") }.toList().toTypedArray()
     Args.parseArgs(args)
+
+    Args.enableModes(Mode.SIMPLE)
```

Comment:
User can set required modes themselves, I do not think we need to restrict them here without a strong reason (like for D8)

Reply:
removed `Args.enableModes(Mode.SIMPLE)` for browser case - fixed.